landbosse/tests/model/test_SitePreparationCost.py

```diff
@@ -13,28 +13,30 @@ class TestSitePreparationCost(TestCase):
     def setUp(self):
```

Comment:
Shouldn't the test case data be set for utility? Maybe we're just planning to go back and revise all of these later. If so, please add an issue on GitHub so we don't forget to do that.

Reply:
@eberlea I think setting this as an issue would work best. Besides, next step after the PRs are approved and merged will focus heavily on testing (based on my team email earlier today). And the testing will cover both utility and DW scale.